wchat/screens/ChatList/index.js

```diff
@@ -1,16 +1,44 @@
 import React, { Component } from 'react';
-import chats from '../../shared/assets/contacts.json'
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux'
 import add_chat_icon from '../../shared/assets/add-chat.png';
 import ContactTable from '../shared/components/ContactTable/index.js'
+import { actionCreators as chatsActions } from '../../redux/chats/actions.js'
 
-export default class ChatListContainer extends Component {
+class ChatListContainer extends Component {
+  componentWillMount() {
+    this.props.dispatch(chatsActions.fetchChats())
+  }
+  onSelect = ({id}) => {
+    this.props.navigation.navigate('Chat', {id: id })
+  }
+  onAdd = () => {
+    console.log("onAdd")
+  }
   render() {
     return (
       <ContactTable
-        data={chats}
+        data={this.props.chats}
         addIcon={add_chat_icon}
-        onAdd={()=>{}}
+        onSelect={this.onSelect}
+        onAdd={this.onAdd}
         />
     );
   }
 }
+
+ChatListContainer.defaultProps = {
+  chats: []
```

Comment:
if default state in reducer is an empty array, you dont need declare it like default prop